autoprotocol/instruction.py

```diff
@@ -1385,9 +1388,19 @@ class Provision(Instruction):
 
     """
 
-    def __init__(self, resource_id, dests):
+    def __init__(self, resource_id, dests, measurement_mode="volume"):
+        if measurement_mode not in PROVISION_MEASUREMENT_MODES:
+            raise RuntimeError(
```

Comment:
add `RuntimeError` to the "Raises" section of this function's docstring

Reply:
Updated `Raises` section